test/test_metropolis.py

```diff
@@ -175,12 +180,12 @@ def test_metropolis_reproducible() -> None:
 
     # reinitialize proposal generator for each model
     proposal_generator = np.random.default_rng(seed=12345)
-    proposal_fn = lambda theta: proposal_generator.normal(loc=theta, scale=4)  # type: ignore
+    proposal_fn = lambda theta: proposal_generator.normal(loc=theta, scale=4)
```

Comment:
Removing the `# type: ignore` here generates an error from `mypy --strict` on my command line.

Reply:
I removed these because mypy was generating an error for "unnecessary type: ignore comment". I have mypy 1.4.1 and numpy 1.23.2, which seems to agree with what the CI picks up

Reply:
K. Looks like it was a mypy issue; I had an out of date version and after updating I no longer get the error.